GDJS/tests/tests/Extensions/testruntimeobject.js

```diff
@@ -40,6 +40,13 @@
     this.hitBoxesDirty = true;
   }
 
+  setCustomZOrder(zorder) {
```

Comment:
Was it needed to add this method for this feature?

Reply:
For some reason, setZOrder with a TestRuntimeObject triggered an error (`cannot read _parentID of undefined`). I suspected it was because the method `getRendererObject` of `TestRuntimeObject` does not return `undefined` (which is the default in RuntimeObject) and setting it to return undefined broke a test in another file...

Reply:
You encountered an error when running tests with your modifications?

Reply:
With my modifications, I had to make my object wider, so I had to use TestRuntimeObject, and changing the class of the object used in the test made it break at the setZOrder method

Reply:
> setting it to return undefined broke a test in another file

Any chance you can share this other test/quickly check why it's failing?

I think in theory we should return undefined, but not sure why we returned an empty object (which looks dodgy).

Reply:
I wonder if we should rework this test to work with a `TestRuntimeObjectWithFakeRenderer`? and have `TestRuntimeObject` return null as the renderer?